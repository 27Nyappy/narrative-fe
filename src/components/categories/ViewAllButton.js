import React from 'react';
import PropTypes from 'prop-types';

const ViewAllButton = ({ handleViewClick }) => {
  return (
    <button onClick={handleViewClick}>View All</button>
  );
};

ViewAllButton.propTypes = {
  handleViewClick: PropTypes.func.isRequired
};

export default ViewAllButton;