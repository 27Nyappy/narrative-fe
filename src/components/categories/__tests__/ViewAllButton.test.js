import React from 'react';
import { shallow } from 'enzyme';
import ViewAllButton from '../ViewAllButton';

describe('ViewAllButton component', () => {
  it('renders view all button', () => {
    const wrapper = shallow(<ViewAllButton />);
    expect(wrapper).toMatchSnapshot();
  });
});