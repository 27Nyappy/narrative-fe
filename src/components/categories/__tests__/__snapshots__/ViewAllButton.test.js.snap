// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewAllButton component renders view all button 1`] = `
<button>
  View All
</button>
`;
